src/libponyrt/sched/start.c

```diff
@@ -107,6 +112,16 @@ static int parse_opts(int argc, char** argv, options_t* opt)
     }
   }
 
+  if (opt->noscale)
+  {
```

Comment:
small nit. this can be fixed in another PR. this `{` should be on the preceding line.